dcbr-api/api/serializers.py

```diff
@@ -145,7 +150,27 @@ def create(self, validated_data):
         for renewal_data in renewals_data:
             Renewal.objects.create(registration_number=registration, **renewal_data)
 
-        Operator.objects.create(registration_number=registration, **operator_data)
+        operator = Operator.objects.create(
+            registration_number=registration, **operator_data
+        )
+
+        def dt_converter(o):
+            if isinstance(o, datetime.datetime):
+                o_date = o.date()
+                print(o_date)
+                return o_date.__str__()
+
+        context = {
+            "op": operator,
+            "email_address": operator.email_address,
+            # "registration_date": json.dumps(
+            #     registration.created_timestamp, default=dt_converter
+            # ),
+            # "expiry_date": registration.expiry_date,
+        }
+        print(context)
+
+        # tasks.send_registration_email(context)
```

Comment:
This is now outdated, please refer to https://github.com/bcgov/agri-dcbr/pull/73 for implementing this. I'd suggest we remove this code from this PR and push the changes once my PR is merged and we're ready to incorporate changes, too much noise otherwise.

Reply:
okay, so I'll park this PR, and look at your changes first.